<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFittingSelf</class>
 <widget class="QWidget" name="FormFittingSelf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FittingSelf</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSpinBox" name="spinBoxStart">
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="tBtnToPlot">
       <property name="text">
        <string>to plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnToHex">
       <property name="text">
        <string>to hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnToIntArray">
       <property name="text">
        <string>to int array</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBoxMiddle">
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelTotal">
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QSpinBox" name="spinBoxTotal">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QPushButton" name="btnCalculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="spinBoxEnd">
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelCountStep1">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxCountStep1">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelCountStep2">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxCountStep2">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>358</width>
       <height>157</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelHead">
       <property name="text">
        <string>Head</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFoot">
       <property name="text">
        <string>Foot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditHead"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditFoot"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxStart</tabstop>
  <tabstop>spinBoxMiddle</tabstop>
  <tabstop>spinBoxEnd</tabstop>
  <tabstop>spinBoxCountStep1</tabstop>
  <tabstop>spinBoxCountStep2</tabstop>
  <tabstop>spinBoxTotal</tabstop>
  <tabstop>lineEditHead</tabstop>
  <tabstop>lineEditFoot</tabstop>
  <tabstop>btnCalculate</tabstop>
  <tabstop>tBtnToPlot</tabstop>
  <tabstop>tBtnToHex</tabstop>
  <tabstop>tBtnToIntArray</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
